### PR TITLE
add header

### PR DESCRIPTION
**Pull request by THN91**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "^4.4.1",
+    "jquery": "^3.4.1",
+    "popper.js": "^1.16.1",
```
Comment (FreeNikita) [inline package.json]: Change to npm install react-bootstrap bootstrap

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import $ from 'jquery';
+import Popper from 'popper.js';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```
Comment (FreeNikita) [inline src/index.js]: need only import 'bootstrap/dist/css/bootstrap.min.css';

File: src/components/header/header.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                        data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                    <span className="navbar-toggler-icon"></span>
+                </button>
+            </nav>
+        </div>
+    );
+}
+
+export default Header
```
Comment (FreeNikita) [inline src/components/header/header.js]: rewrite

File: package.json (hunk 1 above)
Comment (THN91) [reply package.json]: fixed

File: src/index.js (hunk 2 above)
Comment (THN91) [reply src/index.js]: fixed

File: src/components/header/header.js (hunk 3 above)
Comment (THN91) [reply src/components/header/header.js]: i tried to fix

**Pull request merged**